frontend-react/src/components/AuthElement.tsx

```diff
@@ -33,11 +34,12 @@ export const AuthElement = ({
     );
     // Dynamically authorize user from single or multiple allowed user types
     const authorizeMemberType = useMemo(() => {
-        if (memberType === MemberType.PRIME_ADMIN) return true; // Admin authentication always
+        if (adminHardCheck || memberType === MemberType.PRIME_ADMIN)
```

Comment:
do we need the second half of this? wouldn't `memberType === MemberType.PRIME_ADMIN` in any situation where adminHardCheck returned true?

Reply:
Yeah, we might be good to take that out for AuthElement -- I kept looking at it today like "Why _is_ that there?"